tests/infrastructure/golden_images/conftest.py

```diff
@@ -42,3 +42,18 @@ def latest_fedora_release_version():
     latest_fedora_version = str(max(versions))
     LOGGER.info(f"Latest Fedora release: {latest_fedora_version}")
     return latest_fedora_version
+
+
+@pytest.fixture(scope="module")
+def storage_class_from_config_different_from_default(available_storage_classes_names):
+    different_storage_class = next(
+        (
+            storage_class_name
+            for storage_class_name in available_storage_classes_names
```

Comment:
You were right that we can't use `available_storage_classes_names`, because this will fail if we only pass one storage class in the matrix (that's what all scheduled jobs do)

Reply:
Currently, all lanes are running with one SC in the matrix, so this will fail. I don't think we should make such change knowing it. 
If you want to have the tests that clones from one sc to another - `storage_class_a` and `storage_class_b` look like a better option.